packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -836,26 +828,24 @@ test.describe('Routing', () => {
 	test('recognizes clicks outside the app target', async ({ page }) => {
 		await page.goto('/routing/link-outside-app-target/source');
 
-		await page.click('[href="/routing/link-outside-app-target/target"]');
+		await page.locator('[href="/routing/link-outside-app-target/target"]').click();
 		await expect(page.locator('h1')).toHaveText('target: 1');
 	});
 
 	test('responds to <form method="GET"> submission without reload', async ({ page }) => {
 		await page.goto('/routing/form-get');
 		expect(await page.textContent('h1')).toBe('...');
 		expect(await page.textContent('h2')).toBe('enter');
-		expect(await page.textContent('h3')).toBe('...');
```

Comment:
why did this have to change? (and below on 858). I wonder if it was newly added and you copied over my file accidentally losing some changes?

Reply:
lol that's exactly what i did. surprised it didn't result in a merge conflict... fixing